.circleci/config.yml

```diff
@@ -10,17 +10,12 @@ defaults: &defaults
   working_directory: /mnt/ramdisk/repo
 
 version: 2.1
-orbs:
-  # https://circleci.com/developer/orbs/orb/cmgriffing/bun-orb
-  bun-orb: cmgriffing/bun-orb@0.0.26
```

Comment:
Did you mean to remove this orb and bun from CI?

Reply:
Yes but I forgot to commit the fix I did instead. 😅